tests/x11/chromium.pm

```diff
@@ -36,6 +36,11 @@ sub run {
     wait_screen_change { send_key 'ctrl-l' };
     type_string "https://html5test.opensuse.org\n";
     assert_screen 'chromium-html-test', 90;
+
+    # check a site with different ssl configuration (boo#1144625)
+    wait_screen_change { send_key 'ctrl-l' };
+    type_string("https://upload.wikimedia.org/wikipedia/commons/d/d0/OpenSUSE_Logo.svg\n");
+    assert_screen 'chromium-opensuse-logo', 90;
```

Comment:
the problem with this, is when wikimedia is not available, so I'd suggest to check first if it's pingable/reachable with curl or something quick, and then run this "step"

Reply:
I would just try it out - if wikimedia seems to be unstable, we can still add that.

Reply:
especially for openQA tests we have setup instances in the domain of opensuse.org, e.g. the one above, html5test.opensuse.org . Maybe you want to check another internally maintained instance?

Reply:
@okurz I dont think this would trigger the bug with internal instances, because IIRC chromium does have a hardcoded list of domains for which it expects certificate transparency to work.